zib/vocabulary/manual/ValueSet-ContactInformation-TelephoneNumbersUse-EmergencyContact.xml

```diff
@@ -19,7 +18,7 @@
             <system value="http://terminology.hl7.org/CodeSystem/v3-AddressUse"/>
             <concept>
                 <code value="EC"/>
-                <display value="Emergencycontact"/>
+                <display value="Emergency contact"/>
```

Comment:
The actual display value is without a: "emergency contact" 
https://terminology.hl7.org/6.1.0/CodeSystem-v3-AddressUse.html

Reply:
Based on ZIB-1436 and the fact that we have manually created subsets of the zib ValueSet NummerSoortCodelijst (in which capitals are used), Ardon and I concluded that it's better to keep the `display` as-is.